packages/floating-point-hex-parser/test/fuzzing/parse.c

```diff
@@ -0,0 +1,24 @@
+#include "stdio.h"
+#include "stdlib.h"
+#include "time.h"
+#include "math.h"
+#include "float.h"
+
+int main(int argc, char *argv[]) {
+	// test id
+	int i = atoi(argv[1]);
+
+	/* Intializes random number generator */
+	srand(time(NULL) + i);
```

Comment:
@maurobringolf you were already passing an id from the test runner, I added it to the time to ensure no collisions (might not be the best solution tho). Do you know what time precision it uses? In my test i looked like seconds.

Reply:
Seems to be seconds, so this makes a lot of sense :+1: I can only guess from a quick google that it is not possible to get a more exact timestamp using C only: https://stackoverflow.com/a/8558644/6374865